### PR TITLE
Documentation - IdentifyPrimaryObjects

### PR DESCRIPTION
**Pull request by AnneCarpenter**
Someone please check accuracy of my new lines 48-50?

### CODE REVIEW

**Comment by mcquin** [comment]
> Someone please check accuracy of my new lines 48-50?

@bethac07 or @karhohs

File: cellprofiler/modules/identifyprimaryobjects.py [hunk 1]
```diff
@@ -42,7 +45,9 @@
 -  The image should be grayscale.
 -  The foreground (i.e, regions of interest) are lighter than the
    background.
--  The image should be 2D, not volumetric.
+-  The image should be 2D. 2D slices of 3D images are acceptable if
+   loaded as such in the **Images** module. For volumetric analysis
```
Comment (bethac07) [inline cellprofiler/modules/identifyprimaryobjects.py]: Can you change `2D slices of 3D images are acceptable if loaded as such in the **Images** module.` to `2D slices of 3D images are acceptable if the image has not been loaded as volumetric in the **NamesAndTypes** module.` ?

Comment (AnneCarpenter) [reply cellprofiler/modules/identifyprimaryobjects.py]: done!

**Comment by AnneCarpenter** [comment]
Not sure the workflow here, I made change Beth suggested but it's still showing red "Changes requested" from the review -not sure how to indicate that the review has been addressed.

**Comment by bethac07** [comment]
It emailed me so that I can come check what you've done and Approve it, which I've now done 😄.   Now just need @mcquin or @0x00b1 to pull.

**Pull request merged**